packages/shopify-checkout/src/client/index.ts

```diff
@@ -134,10 +151,19 @@ export default function createShopifyCheckoutClient({
     return await removeDiscount({ gqlClient, id, queueToken });
   }
 
+  async function query({
+    query,
+    input,
+    queueToken
+  }: CustomQueryParams): Promise<ShopifyResponse<unknown> | void> {
+    return await queryCheckout({ gqlClient, query, input, queueToken });
+  }
```

Comment:
An extension of the feedback in `packages/shopify-checkout/src/client/actions/queryCheckout.ts`:

```suggestion
  async function query({
    query,
    variables
  }: CustomQueryParams): Promise<ShopifyResponse<
    Record<string, unknown>
  > | void> {
    return await queryCheckout({ gqlClient, query, variables });
  }
```